app/tests/containers/Forms/__snapshots__/Form.test.tsx.snap

```diff
@@ -1098,6 +1098,208 @@ exports[`Form should match the snapshot with the electricity and heat form 1`] =
 </form>
 `;
 
+exports[`Form should match the snapshot with the emission form 1`] = `
+<form
+  class="rjsf"
+>
+  <div
+    class="col-12"
+  >
+    <div
+      class="form-row"
+    >
+      <div
+        class="col-12"
+      >
+        <legend>
+          Emissions
+        </legend>
+      </div>
+      <div
+        class="col-12"
+      >
+        <div
+          class="form-row"
+        >
+          <div
+            class="col-md-12 col-12"
+          >
+            <div
+              class="col-12"
+            >
+              <div
+                class="form-row"
+              >
+                <div
+                  class="col-12"
+                >
+                  <legend />
+                </div>
+                <div
+                  class="form-group field field-string  hidden-title form-group col-xl-12 col-lg-12 col-md-12 col-sm-12 col-12"
+                >
+                  <label
+                    class="form-label"
+                    for="root_sourceTypes_0_sourceTypeName"
+                  >
+                    sourceTypeName
+                  </label>
+                  <div
+                    class="row"
+                    style="margin:20px 0 20px 0"
+                  >
+                    <div
+                      class="col-md-4 col-12"
+                    >
+                      <h5
+                        class="jsx-2806087631"
+                        style="text-align:left"
+                      >
+                        Stationary Combustion
```

Comment:
Where does this come from? There is no form data in the test

Reply:
fixed by removing defaults